app/src/main/java/com/pileproject/drive/programming/visual/block/BlockFactory.java

```diff
@@ -38,131 +38,93 @@
     public static final int SEQUENCE = 0;
     public static final int REPETITION = 1;
     public static final int SELECTION = 2;
+    @Deprecated
     public static final int UNDO = 3;
     public static final int LOAD = 4;
 
-    /**
-     * This class can't be created as an instance
-     */
     private BlockFactory() {
     }
 
-    /**
-     * Return class by name
-     *
-     * @param className
-     * @return Class
-     */
     @SuppressWarnings("unchecked")
-    private static <T> Class<T> getClassForName(String className) throws RuntimeException {
+    private static <T extends BlockBase> Class<T> getClassForName(String className) throws RuntimeException {
+
         try {
             return (Class<T>) Class.forName(className);
+
```

Comment:
Don't take my comment seriously and offensive, please.
For consistency you are right.
And topics of the format are bit off from here.
Anyway I'll merge this. Thanks.

incidentally you can witch-hunt with
```
$ grep -ElR "catch \(.*\) {" app | xargs perl -0777 -pe 's/\n(\s*} catch \(.*\) {)\n/$1/g'
```
for making your realm calm and getting rid of "pagans"